pkg/restapi/v1/verifier/controller_test.go

```diff
@@ -43,22 +52,33 @@ var (
 	sampleVCJWT string
 )
 
+// nolint:gochecknoglobals
+var ariesSupportedKeyTypes = []kms.KeyType{
+	kms.ED25519Type,
+	kms.X25519ECDHKWType,
+	kms.ECDSASecp256k1TypeIEEEP1363,
+	kms.ECDSAP256TypeDER,
+	kms.ECDSAP384TypeDER,
+	kms.RSAPS256Type,
+	kms.BLS12381G2Type,
+}
+
 //nolint:gochecknoglobals
 var (
 	verificationChecks = &verifiersvc.VerificationChecks{
-		Credential: &verifiersvc.CredentialChecks{
+		Credential: verifiersvc.CredentialChecks{
```

Comment:
is there a specific reason why Credential is value type while Presentation below is reference?

Reply:
Credential is not optional, Presentation is optional